test/integration/default/serverspec/server_spec.rb

```diff
@@ -1,3 +1,17 @@
-require 'serverspec'
+require 'spec_helper'
 
-set :backend, :exec
+describe 'postfix' do
+  it_behaves_like 'postfix'
+end
+
+describe file '/etc/postfix/main.cf' do
+  [
+    '# Configured as client',
+    /myorigin = \$mydomain/,
+    'relayhost = \[smtp.osuosl.org\]:25',
+    'smtpd_use_tls = no',
+    'smtp_use_tls = no',
+  ].each do |line|
+    its(:content) { should match(line) }
+  end
+end
```

Comment:
Can you also add a check here for the CA cert if the platform is Debian and not otherwise?